core/src/main/scala/kafka/tools/DumpLogSegments.scala

```diff
@@ -430,6 +431,11 @@ object DumpLogSegments {
       .describedAs("size")
       .ofType(classOf[java.lang.Integer])
       .defaultsTo(5 * 1024 * 1024)
+    val maxBytesOpt = parser.accepts("max-bytes", "Limit the amount of total batches in bytes avoiding reading the whole file(s).")
```

Comment:
It would be useful to make it clear this option only applies to the .log files.

Reply:
makes sense, added!